explainaboard/metric.py

```diff
@@ -91,12 +90,16 @@ def default_name(cls) -> str:
         """Returns the default name of the metric."""
         ...
 
-    def __init__(self, name: str = None):
+    def __init__(
```

Comment:
Suggestion: could we replace these individual variables with a class `MetricConfig`? The reason why I suggest this is that then the MetricConfig class could be (1) passed directly to the various metric computing functions, and (2) saved in a database or report, allowing the evaluation setup to be reproducible.

Reply:
Sounds good, I will make it an issue for another future PR.